### PR TITLE
Add recipe for google-auth-httplib2

### PR DESCRIPTION
**Pull request by parthea**
For https://pypi.python.org/pypi/google-auth-httplib2/0.0.2

### CODE REVIEW

**Comment by parthea** [comment]
This is waiting on conda-forge/staged-recipes#3099 . Nothing to do here until conda-forge/staged-recipes#3099 is ready.

**Comment by parthea** [comment]
All checks passed. This is ready for review.

cc @jakirkham

File: recipes/google-auth-httplib2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    discouraged. This library is intended to help existing users of oauth2client migrate to
+    google-auth.
+  doc_url: https://google-auth.readthedocs.io/en/latest/
+  dev_url: https://github.com/GoogleCloudPlatform/google-auth-library-python
+
+extra:
+  recipe-maintainers:
+    - parthea
+    - jreback
+    - tswast
```
Comment (jakirkham) [inline recipes/google-auth-httplib2/meta.yaml]: Are both of you ok being maintainers here, @jreback and @tswast?

File: recipes/google-auth-httplib2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  imports:
+    - google_auth_httplib2
+
+about:
+  home: https://github.com/GoogleCloudPlatform/google-auth-library-python
+  license: Apache-2.0
+  license_family: Apache
+  summary: 'Google Authentication Library httplib2 transport'
+  description: This library provides an httplib2 transport for google-auth. Note that httplib has
+    lots of problems such as lack of threadsafety and and insecure usage of TLS. Using it is highly
```
Comment (jakirkham) [inline recipes/google-auth-httplib2/meta.yaml]: Duplicate "and".

Comment (parthea) [inline recipes/google-auth-httplib2/meta.yaml]: Done.

File: recipes/google-auth-httplib2/meta.yaml (hunk 1 above)
Comment (jreback) [inline recipes/google-auth-httplib2/meta.yaml]: ok

Comment (tswast) [inline recipes/google-auth-httplib2/meta.yaml]: yes

**Comment by parthea** [comment]
@jakirkham Please review at your earliest convenience

**Comment by jakirkham** [comment]
Thanks @parthea.

**Pull request merged**